app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/summary/SummaryProductFragment.java

```diff
@@ -420,8 +443,60 @@ public void onClick(View textView) {
                     CustomTabsIntent customTabsIntent = CustomTabsHelper.getCustomTabsIntent(getContext(), customTabActivityHelper.getSession());
                     CustomTabActivityHelper.openCustomTab(SummaryProductFragment.this.getActivity(), customTabsIntent, nutritionScoreUri, new WebViewFallback());
                 });
+                nutritionLightsCardView.setVisibility(View.VISIBLE);
+                Nutriments.Nutriment fatNutriment = nutriments.get(Nutriments.FAT);
```

Comment:
nutriments can be null. Please add NPE check when used.

Reply:
okay

Reply:
done.